app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -5,17 +5,15 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.BaseAdapter;
```

Comment:
> Bonus question: Why are all of the server operations in MessagesActivity wrapped in an AsyncTask if they end up in a Retrofit enqueue call anyway? I think enqueue can be called from the UI thread.

I dunno actually. I can remember that I one time got the "No Network On Main Thread Exception" thingy and then I started wrapping everything into async tasks. Earlier the app used a different api client maybe that one behaved differently. 

But yeah, this was my first android app, so I bet there is a lot stuff that could be improved :D.